src/Extension/OrderedButtonExtension.php

```diff
@@ -24,6 +24,6 @@ class OrderedButtonExtension extends AbstractOrderedExtension
     */
     public function getExtendedType()
     {
-        return 'button';
+        return 'Symfony\Component\Form\Extension\Core\Type\ButtonType';
```

Comment:
I always use this kind of solutions myself, but it's not really bullet-proof, right? It's circumstantial evidence. Wouldn't there be a way to do proper version comparison?